Find a functionally equivalent code:android.media.MediaDrm.getProvisionRequest:COMMENT
<android.media.MediaDrm: ProvisionRequest getProvisionRequest()>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * A provision request/response exchange occurs between the app and a provisioning
 * server to retrieve a device certificate.  If provisionining is required, the
 * EVENT_PROVISION_REQUIRED event will be sent to the event handler.
 * getProvisionRequest is used to obtain the opaque provision request byte array that
 * should be delivered to the provisioning server. The provision request byte array
 * is returned in ProvisionRequest.data. The recommended URL to deliver the provision
 * request to is returned in ProvisionRequest.defaultUrl.
 */

@@ -1 +1,3 @@
-
+{
+    return getProvisionRequestNative(CERTIFICATE_TYPE_NONE, "");
+}

